ultrasonic/src/main/res/values/arrays.xml

```diff
@@ -236,5 +236,40 @@
         <item>@string/settings.playback.bluetooth_a2dp</item>
         <item>@string/settings.playback.bluetooth_disabled</item>
     </string-array>
+    <string-array name="languageNames" translatable="true">
+        <item>@string/language.default</item>
+        <item>@string/language.zh</item>
```

Comment:
As we have two chinnese language I think that is better to use `zh_cn` Chinese (China) and `zh_tw` Chinese (Taiwan)